Fix missing defaults/loom CLI wrapper causing test failures

## Problem

Main branch tests are failing because `defaults/loom` CLI wrapper doesn't exist:

```
FAILED tests/test_installation_verification.py::TestWrapperScriptRouting::test_cli_wrapper_health_routes_to_python
AssertionError: loom CLI wrapper not found
assert False
  where False = PosixPath('/Users/rwalters/GitHub/loom/defaults/loom').exists
```

This pre-existing failure is blocking shepherd work on unrelated issues (see #2006).

## Impact

- All shepherds running `pnpm check:ci:lite` see baseline failures
- Test comparison logic gets confused by different failure types
- PR #2041 was blocked despite implementing its feature correctly

## Solution

Either:
1. Create the missing `defaults/loom` CLI wrapper
2. Or update/remove the test if the wrapper was intentionally removed

## Acceptance Criteria

- [ ] `pnpm check:ci:lite` passes on main
- [ ] `defaults/loom` exists OR test is updated

## References

- Discovered during shepherd run for #2006
- Blocking: #2041

--- a/loom-tools/tests/test_installation_verification.py
+++ b/loom-tools/tests/test_installation_verification.py
@@ -203,7 +203,7 @@ class TestWrapperScriptRouting:
         self, defaults_dir: pathlib.Path
     ) -> None:
         """The 'loom' CLI wrapper should route health to Python."""
-        cli_wrapper = defaults_dir / "loom"
+        cli_wrapper = defaults_dir / ".loom" / "bin" / "loom"
         assert cli_wrapper.exists(), "loom CLI wrapper not found"
 
         content = cli_wrapper.read_text()

